SPECS/grpc/grpc.spec

```diff
@@ -1,30 +1,39 @@
 Summary:        Open source remote procedure call (RPC) framework
 Name:           grpc
 Version:        1.42.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/System
 URL:            https://www.grpc.io
 Source0:        https://github.com/grpc/grpc/archive/v%{version}/%{name}-%{version}.tar.gz
-
+Source1:        %{name}-%{version}-submodules.tar.gz
```

Comment:
in term of auto-upgrade it would be easier to incorporate the submodules into the Source0 tarball (1 tarball instead of using 2 tarballs)

Reply:
I actually did that for my first PR but I was suggested by Pawel and Olivia to create a second source for the git submodules.